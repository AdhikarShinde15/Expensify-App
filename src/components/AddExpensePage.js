import React from "react";

const AddExpensePage = () => (
    <div>
        Add Expense PAge
    </div>
);

export default AddExpensePage
