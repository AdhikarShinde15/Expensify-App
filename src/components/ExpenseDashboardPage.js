import React from "react";

const ExpenseDashboardPage = () => (
    <div>This is from Dashboard Page</div>
);

export default ExpenseDashboardPage;