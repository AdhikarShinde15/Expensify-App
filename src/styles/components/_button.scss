.btn {
    background: $blue;
    border: none;
    outline: none;
    display: inline-block;
    padding: $s-size;
    font-weight: 300;
    font-size: $font-size-large;
    color: white;
    line-height: 1;
    text-decoration: none;
}

.btn--link {
    background: none;
}