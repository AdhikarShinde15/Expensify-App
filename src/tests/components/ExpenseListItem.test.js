import React from "react";
import { ExpenseListItem } from "../../components/ExpenseListItem";
import { shallow } from "enzyme";
import expenses from "../fixtures/expenses";

test('Should Render Expense List Item With one of the Item', () => {
    const wrapper = shallow(<ExpenseListItem {...expenses[1]}/>)
    expect(wrapper).toMatchSnapshot();
})