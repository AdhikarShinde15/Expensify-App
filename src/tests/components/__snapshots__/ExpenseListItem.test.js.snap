// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Render Expense List Item With one of the Item 1`] = `
<div>
  <Link
    to="/edit/2"
  >
    <h3>
      Rent
    </h3>
  </Link>
  <p>
    109500
     -- 
    -345600000
  </p>
</div>
`;
